src/parser.js

```diff
@@ -3,11 +3,17 @@
 const handle = require('./handle.js');
 const templatizer = require('./templatizer.js');
 const validate = require('./configValidation.js');
+const configDefault = require('./template/configDefault.json');
+const noDocsError = new Error('No documentation passed');
 
-const parser = (help, config) => {
-    config = validate(config);
```

Comment:
Assign it to a local variable, functions with side effects are less readable.

Reply:
It also confuses that `validate` checks and modify the object.

Reply:
@xferra done